### ISSUE TITLE
Openkeychain Android-Password-Store passwordstore Yubikey

### ISSUE THREAD

**Issue by werttig** [zeapo/Android-Password-Store#503]
Hello pgp-community,

I use a yubikey nfc on passwordstore v1.7.3 under arch which works perfect. Cloning the corresponding git repository with Android-Password-Store v1.3.2 (Samsung galaxy s5+, Lineageos 16, OpenKeychain 5.2) works as well.

The problem is that trying to decrypt the passwords on Android-Password-Store v1.3.2, generated on passwordstore v1.7.3, gives an error message. The other way is working, meaning that I can decrypt passwords on arch, passwordstore v1.7.3 and Android-Password-Store v1.3.2, which where generated on lineageos 16, Android-Password-Store v1.3.2.

I use subkeys for all key types (encryption, authentication,...) and the encryption type key is the one in .gpg-id. The fingerprint of the public (Masterkey) is the one selected on Android-Password-Store and the corresponding private key is not stored on the yubikey.

Has anyone whith experience in this context an idea of how to get this fixed and where to start investigating or do you need more information?

**Comment by r3mix-ch** [comment]
Hi,

I've got the same issue, it seems. I've tried with bothe password Store v 1.3.2 from the Play Store and F-Droid, as well as v 1.3.1 from F-Droid. Same issue. OpenKeychain seems to have a problem to enable Password Store to use the keys store in OpenKeychain. I use the my private key id, not the subkey for encryption. I use the Yubikey's smart card functionality, so the keys are not physically present on the device, only the stubs.
